index.bs

```diff
@@ -4617,6 +4617,7 @@ interface IDBCursor {
   readonly attribute IDBCursorDirection direction;
   readonly attribute any key;
   readonly attribute any primaryKey;
+  readonly attribute IDBRequest request;
```

Comment:
In addition to this IDL addition, please add a non-normative description to the  `domintro` block just below it (i.e. a new `dt`/`dd` pair), aimed at web developers and describing usage with need-to-know information.